Library/Formula/cccc.rb

```diff
@@ -0,0 +1,58 @@
+class Cccc < Formula
+  desc "C and C++ Code Counter"
+  homepage "http://sourceforge.net/projects/cccc/"
+  url "https://downloads.sourceforge.net/project/cccc/cccc/3.1.4/cccc-3.1.4.tar.gz"
+  sha256 "27b3eca9a83a66799332363a80cc4bcd6db9869eddbda1a59a25cdace3ff4dbf"
+
+  # reported to upstream: https://sourceforge.net/p/cccc/bugs/55/
+  patch :DATA
```

Comment:
Just to be a bit neater it might be nice to turn these into an `inreplace` in `def install`, i.e.

```ruby
def install
  # Bug report link
  inreplace "cccc_tbl.cc", "erase(value_iterator);", "map_t::erase(value_iterator);"
  inreplace "makefile", 'su root -c "make -f install.mak"', "make -f install.mak"

  # Bug report link
  ENV.deparallelize
end
```